Dockerfile

```
@@ -74,6 +75,37 @@ RUN \
     apt-get clean -qy &&\
     rm -rf /var/lib/apt/lists/*
 
+# Multi-stage build: Build static assets
+# This allows us to not include Node within the final container
+FROM node:20 as node_modules_go_brrr
+
+RUN mkdir /app
+
+RUN mkdir -p  /app
+WORKDIR /app
+COPY --from=base /var/www/html/Lychee /app
+
+# Use yarn or npm depending on what type of
+# lock file we might find. Defaults to
+# NPM if no lock file is found.
+# Note: We run "production" for Mix and "build" for Vite
+RUN \
+    npm ci --no-audit \
+    npm run build
+
+# From our base container created above, we
+# create our final image, adding in static
+# assets that we generated above
+FROM base
+
+# Packages like Laravel Nova may have added assets to the public directory
+# or maybe some custom assets were added manually! Either way, we merge
+# in the assets we generated above rather than overwrite them
+COPY --from=node_modules_go_brrr /app/public /var/www/html/Lychee/public-npm

```

```suggestion
COPY --from=node_modules_go_brrr --chown=www-data:www-data /app/public /var/www/html/Lychee/public-npm
```
Might mean you can drop the `chown ` later?

---

I am not familiar with that part.

---

Can add in the "may improve later" pile.